### PR TITLE
[ADD] Ask for confirmation when copying records

### PR DESCRIPTION
**Pull request by StefanRijnhart**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Microsoft syndrome? 😆 

No, it's common to press there instead of "Delete", so good idea!

File: web_copy_confirm/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Show confirmation dialogue before copying records",
+    "version": "8.0.1.0.0",
+    "author": "Dynapps,Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "category": "Tools",
+    "depends": [
+        'web',
+    ],
```
Comment (simahawk) [inline web_copy_confirm/__openerp__.py:14]: 'website': 'https://github.com/OCA/web' ?

**Review by simahawk — APPROVED** [review]

**Review by legalsylvain — APPROVED** [review]
Thanks.

File: web_copy_confirm/__openerp__.py (hunk 1 above)
Comment (StefanRijnhart) [reply web_copy_confirm/__openerp__.py:14]: Thanks, added now!

**Review by moylop260 — APPROVED** [review]
:+1:
Tested on runbot

**Pull request merged**